doc/api/buffer.md

```diff
@@ -3415,7 +3415,13 @@ added: v8.2.0
 * {integer} The largest size allowed for a single `Buffer` instance.
 
 On 32-bit architectures, this value currently is 2<sup>30</sup> - 1 (~1GB).
-On 64-bit architectures, this value currently is 2<sup>31</sup> - 1 (~2GB).
+On 64-bit architectures, this value currently is
+
+* for v12: 2<sup>31</sup> - 1 (~2GB)
+* for v14: 2<sup>32</sup> - 1 (~4GB)
+* for v15: 2<sup>32</sup>(~4GB)
```

Comment:
Could we instead use a `<!-- YAML` to document those changes, and only have the value corresponding to the current value in prose?
```suggestion
On 64-bit architectures, this value currently is 2<sup>32</sup> (~4GB).
```